instrumented/aboutUs/Info/index.ts

```diff
@@ -0,0 +1,33 @@
+function cov_1lc5k6fmi() {
+  const path =
+    'C:\\react\\first\\helgaathame-JSFE2022Q3\\react.component\\src\\aboutUs\\Info\\index.ts';
```

Comment:
this definitely wont work for everybody

Reply:
if some files can be generated from initial code - leave instruction in README file and delete those autogenerated files